apple/swift-docc-render
# Make sure we show only tags, that match the currently filtered items

Bug/issue #, if applicable: 89482723

## Summary

This PR further limits the available tags, to show only those tags, that have matches in the current list of renderable items.

## Dependencies

NA

## Testing

Steps:
1. Check the tags
2. Start typing in the navigator to filter out items
3. Assert tags are filtered with the list of items as well.

## Checklist

Make sure you check off the following items. If they cannot be completed, provide a reason.
